src/frontend/screens/MapScreen/track/SaveTrackScreen.tsx

```diff
@@ -55,7 +51,7 @@ export const SaveTrackScreen = () => {
 
   const handleDiscard = () => {
     closeSheet();
-    navigation.navigate(TabName.Map);
+    navigation.navigate('Map');
```

Comment:
Your navigation action needs to be a reset here as well (same comment as in `saveTrackButton`). The user needs to be prevented from going back to this screen